src/components/profile.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable */
```

Comment:
- [ ] Linting is a great tool so it is better to avoid disabling them to have some clean code